build/images/scripts/install_cni_windows.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = "Stop";
```

Comment:
You said you will change the file name to "InstallCNIWindows.ps1"? BTW, I saw some articles saying the file names should be <Verb>-<Noun>. In this case, it is Install-CNIWindows.ps1?

Reply:
Yes, the recommended name should be "Install-CNIWindows.ps1". But seems only windows official commands following the rule.

Anyway, it's always good to follow the official recommendation. I will rename the windows scripts like that.